pkg/helm/charts/tests/securedclusterservices/testdata/helmtest/scanner-v4-image.test.yaml

```diff
@@ -0,0 +1,78 @@
+defs: |
+  def scannerV4Indexer:
+    container(.deployments.scanner-v4-indexer; "indexer");
+  def scannerV4DB:
+    container(.deployments.scanner-v4-db; "db");
+values:
+  imagePullSecrets:
+    allowNone: true
+  scannerV4:
+    disable: false
+
+tests:
+- name: "test image settings"
+  tests:
+  - name: "custom registry for scanner-v4 images"
+    set:
+      image.scannerV4.registry: "scanner-v4-registry.example.com"
+      image.scannerV4.registry: "scanner-v4-db-registry.example.com"
+    expect: |
+      scannerSlim | .image | assertThat(startswith("scanner-registry.example.com/scanner-slim:"))
```

Comment:
Given the `set` configuration above, shouldn't the expect clauses reference scanner v4?